pkg/management/component_test.go

```diff
@@ -3217,3 +3217,55 @@ func TestUpdateRealmCustomConfiguration(t *testing.T) {
 		assert.NotNil(t, err)
 	}
 }
+
+func TestCreateShadowUser(t *testing.T) {
+	var mockCtrl = gomock.NewController(t)
+	defer mockCtrl.Finish()
+	var mockKeycloakClient = mock.NewKeycloakClient(mockCtrl)
+	var mockEventDBModule = mock.NewEventDBModule(mockCtrl)
+	var mockConfigurationDBModule = mock.NewConfigurationDBModule(mockCtrl)
+	var mockLogger = mock.NewLogger(mockCtrl)
+
+	var managementComponent = NewComponent(mockKeycloakClient, mockEventDBModule, mockConfigurationDBModule, mockLogger)
+
+	var accessToken = "TOKEN=="
+	var username = "test"
+	var realmName = "master"
+	var userID = "41dbf4a8-32a9-4000-8c17-edc854c31231"
+	var provider = "provider"
+
+	// Create shadow user
```

Comment:
You can use t.Run(...)
(see in other tests how it works)

Reply:
fixed